_posts/interactive/disorder-outbreak.html

```diff
@@ -0,0 +1,55 @@
+<section id="disorder-outbreak" class="interactive fullWidthFeatureContent">
+  <div id="disorder-outbreak__intro" class="interactive__intro">
+
+    <header class="disorder-outbreak__header">
+      <h3 class="disorder-outbreak__header-title">
```

Comment:
It's going to depend on how the rest of the page is laid out, but if this isn't part of a section that is started by an `<h2>`, then this should be an `<h2>` to maintain proper page hierarchy.